update-resources.sh

```diff
@@ -125,6 +125,21 @@ function add_juicypotato() {
   add-to-list "JuicyPotato.exe,https://github.com/ohpe/juicy-potato,https://github.com/ohpe/juicy-potato"
 }
 
+function add_printspoofer() {
+  info "Downloading PrintSpoofer"
+  chkfs "./windows/"
+  wget -O ./windows/PrintSpoofer32.exe "https://github.com/itm4n/PrintSpoofer/releases/download/v1.0/PrintSpoofer32.exe"
+  wget -O ./windows/PrintSpoofer64.exe "https://github.com/itm4n/PrintSpoofer/releases/download/v1.0/PrintSpoofer64.exe"
```

Comment:
Prefer using a 'latest' reference instead of a fixed tag, like this:
https://github.com/itm4n/PrintSpoofer/releases/latest/download/PrintSpoofer64.exe

Reply:
Got it !